src/ontology/mondo-edit.obo

```diff
@@ -554046,15 +554046,21 @@ is_a: MONDO:0700092 {source="OMIM:619972"} ! neurodevelopmental disorder
 [Term]
 id: MONDO:0859267
 name: tumor predisposition syndrome 2
+def: "Tumor predisposition syndrome-2 (TPDS2) is an autosomal recessive cancer predisposition syndrome characterized by the onset of various types of tumors or malignancies in young adulthood. The most common clinical manifestations include acute myeloid leukemia (AML), myelodysplastic syndrome, colorectal adenomatous polyposis and carcinoma, and uveal melanoma." [OMIM:619975]
```

Comment:
will update the definition so it doesn't repeat the label at the beginning of it.

Reply:
I think we can also add @katiermullen's orcid for this definition.